C++/0. Setup.md

```diff
@@ -2,7 +2,12 @@
 
 ## Start
 
-So, in C++ files, there are some things you need to add at the start of the document. `#include <iostream>` includes a package in C++ which lets you use `cout` and `cin` which are the keywords for character output and character input. In some beginner programmer's programs, you may see `using namespace std;` at the starting too, which lets you use a lot of functions without needing to write `std::functionName` all the time. This is useful, but it sometimes includes functions you don't need and you didn't know were there, resulting with you creating a function with the same name which is used in a function in the namespace. The `int main() {}` is where you will put your code. Your code goes inside the curly braces `{}`. This creates a function called main, which is automatically run when you run the file, and the code inside the braces is what will be run.
```

Comment:
> So, in C++ files, there are some things you need to add at the start of the document.

This was quite blatently wrong. It is not necessary to use any headers at all, but importing or including other code, especially the standard library (on non-embedded environments), is a necessity for most applications.

Reply:
noice!